app/Generators/PullQuote.php

```diff
@@ -5,9 +5,55 @@
 class PullQuote extends TwigRenderer implements GeneratorInterface
 {
 	public function get( $widget ) {
+		$text = $this->remove_quotes(
+			$this->replace_smart_quotes($widget['text'])
+		);
+
 		return $this->renderer->render(
 			'pull-quote.twig',
-			[ 'text' => $widget['text'] ]
+			[ 'text' => $text ]
+		);
+	}
+
+	private function remove_quotes( $text ) {
+		if ( substr( $text, 0, 1) === '"') {
+			$text = substr( $text, 1, strlen( $text ) );
+		}
+
+		if ( substr( $text, -1, 1) === '"') {
+			$text = substr( $text, 0, -1 );
+		}
+
+		return $text;
+	}
+
+	private function replace_smart_quotes( $text ) {
```

Comment:
Does FBIA not support smart quotes?

Reply:
It does, I am just tyding and removing any quotation from the phrase, the smart quotes are added in the template.